ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,344 @@
+#!/bin/sh
```

Comment:
Change it to /bin/bash. Otherwise, on Mac below statement gives error "/system_init_commands.sh: line 31: (15365589493N: value too great for base (error token is "15365589493N")".

$(date +%s%3N)

Reply:
Review Comment Incorporated in code